### DISCUSSION TITLE
Author / View events from Entries

### DISCUSSION THREAD

**Discussion by alexr8** [solspace/craft-calendar#148] [Feature Requests]
As far as I know this isn't possible already (and there may be good reasons for not doing this), but it would be useful to be able to view the 'Events' tab from Entries in the Craft admin. For our clients, having all the bits they edit regularly in one place would help streamline their workflow.

I wondered if there might be a way to hook into the 'Custom Source' functionality in Craft 4, or have it as a toggle in the settings for Calendar to show 'Events' in the sidebar in Entries. Be interested to see if there's any appetite for this :)

**Comment by zjr** [comment]
I'm also interested in this. I'm new to Craft so I wonder if I'm just missing something, but not being able to access Events through the Entries UI or link to them through an Entries type field is a prohibitive problem for me that requires I skip the plugin.

**Reply by laurabennett** [reply]
Calendarize can be used within Events although there are other drawbacks.

**Comment by kjmartens** [comment]
This is something that is not possible in Craft, as Calendar events are their own Element type, and therefore need to be displayed separately from Craft Entries element type. 🙂
